docs/gettingstarted/installation.rst

```diff
@@ -0,0 +1,37 @@
+Installation
+============
+
+nested-pandas is available to install with pip, using the "nested-pandas" package name:
+
+.. code-block:: bash
+
+    pip install nested-pandas
```

Comment:
It might be best to have this be:
! pip install nested-pandas
or
% pip install nested-pandas